### PR TITLE
[runSofa] Removed obsolete options

### PR DESCRIPTION
**Pull request by fredroy**
- simulationType is DAG in any case
~~- interactive is not used~~
- testMode is... not useful ? (mileage can vary though)

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Review by bakpaul — APPROVED** [review]

**Comment by damienmarchal** [comment]
Option -i is still used by RealGui:
```cpp
void RealGUI::parseOptions()
{

    if (mArgumentParser) {
        mArgumentParser->getValueFromKey("interactive", m_enableInteraction);
        mArgumentParser->getValueFromKey("msaa", m_viewerMSAANbSampling);

        if (m_enableInteraction)
            msg_warning("runSofa") << "you activated the interactive mode. This is currently an experimental feature "
                                      "that may change or be removed in the future. ";
    }
}
```

**Comment by fredroy** [comment]
[ci-build]

File: applications/projects/runSofa/Main.cpp [hunk 1]
```diff
@@ -297,18 +277,22 @@ int main(int argc, char** argv)
         "f,formatting",
         "select the message formatting to use (auto, clang, sofa, rich, test)"
     );
+    argParser->addArgument(
```
Comment (bakpaul) [inline applications/projects/runSofa/Main.cpp:280]: Why moving it around ?

Comment (fredroy) [reply applications/projects/runSofa/Main.cpp:280]: I just wanted to gather variables which are not used in runSofa itself

**Comment by fredroy** [comment]
[ci-build][with-all-tests][force-full-build]

**Review by alxbilger — APPROVED** [review]

**Pull request merged**